Dockerfile

```
@@ -1,21 +1,21 @@
-FROM nimlang/nim:2.2.4-ubuntu-slim
+FROM ubuntu:latest
 
-ENV DEBIAN_FRONTEND=noninteractive
-
-RUN apt-get update && \
-    apt-get install -y jq build-essential git libgtk-3-dev libwebkit2gtk-4.0-dev libmpfr-dev && \
-    apt-get purge --auto-remove && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN git clone https://github.com/arturo-lang/arturo.git
-
-RUN cd arturo && ./build.nims build --install --log && \
-    cd .. && rm -rf arturo
-
-ENV PATH="/root/.arturo/bin:${PATH}"
-
-RUN arturo --package install unitt 2.0.1
+RUN apt-get update \
+ # Install required build and runtime dependencies, cannot use `--no-install-recommends`
+ && apt-get install -y jq wget unzip libmpfr6 libwebkit2gtk-4.1-0 libpcre3 \
+ # Fetch pre-built nightly release and install it
+ && wget --quiet --output-document arturo.zip https://github.com/arturo-lang/nightly/releases/download/2025-07-05/arturo-nightly.2025-07-04-amd64-linux-full.zip \
+ && unzip -d /usr/local/bin arturo.zip arturo \
+ && rm arturo.zip \
+ # Install unitt test framework
+ && arturo --package install unitt 2.0.1 \
+ # Clean up apt-get and build dependencies: python3 was installed but not necessary for runtime
+ && apt-get remove -y --purge wget unzip python3 \
+ && apt-get purge --auto-remove -y \
```

`apt-get remove --purge` is identical to `apt-get purge`.

I wonder if the two purge commands can be merged (if it makes any difference):

```suggestion
 && apt-get purge --auto-remove -y wget unzip python3 \
```

---

Done, force pushed. It changes nothing to the size so this probably works the same, thanks!